### PR TITLE
🎨Image pulling progress: use overall size (Part 2)

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- using the overall size of all images, this brings back the old behavior of parallelized pulling of images (improved performance when pulling multiple images)
- ensure progress reporting kind of works in case where it is not possible to retrieve docker image layer information prior to pulling
- tests consistency
- test when no layer information is available

@odeimaiz next PR will bring the current size/total size

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by bisgaard-itis — APPROVED** [review]
Cool, thanks

**Review by odeimaiz — COMMENTED** [review]
👍

**Review by odeimaiz — APPROVED** [review]

**Review by pcrespov — APPROVED** [review]
very nice

File: packages/service-library/src/servicelib/fastapi/docker_utils.py [hunk 1]
```diff
@@ -114,31 +114,31 @@ async def pull_images(
...
-                registry_settings,
-                pbar,
-                log_cb,
-                (
-                    image_layer_info
-                    if isinstance(image_layer_info, DockerImageManifestsV2)
-                    else None
-                ),
-            )
+        await asyncio.gather(
```
[the remark this answers is not in the document]
Comment (sanderegg) [reply packages/service-library/src/servicelib/fastapi/docker_utils.py:127]: this is anyway limited by the docker engine

**Comment by sanderegg** [comment]
> very nice

not really but makes the job

**Pull request merged**